web_src/less/_repository.less

```diff
@@ -1,28 +1,4 @@
 .repository {
-  .popup.commit-statuses {
-    // we had better limit the max size of the popup, and add scroll bars if the content size is too large.
-    // otherwise some part of the popup will be hidden by viewport boundary
```

Comment:
Can this cause a regression?

Like, if the PR is large enough so the popup can appear above or below the viewport.

Reply:
I searched the whole project and `commit-statuses` is used nowhere along `popup`.